mulighetsrommet-api/src/main/kotlin/no/nav/mulighetsrommet/kafka/amt/AmtDeltakerV1TopicConsumer.kt

```diff
@@ -0,0 +1,63 @@
+package no.nav.mulighetsrommet.kafka.amt
+
+import kotlinx.serialization.json.JsonElement
+import kotlinx.serialization.json.decodeFromJsonElement
+import no.nav.common.kafka.consumer.util.deserializer.Deserializers.uuidDeserializer
+import no.nav.mulighetsrommet.api.repositories.DeltakerRepository
+import no.nav.mulighetsrommet.domain.dbo.DeltakerDbo
+import no.nav.mulighetsrommet.domain.dbo.Deltakerstatus
+import no.nav.mulighetsrommet.kafka.KafkaTopicConsumer
+import no.nav.mulighetsrommet.kafka.amt.AmtDeltakerV1Dto.Status
+import no.nav.mulighetsrommet.kafka.serialization.JsonElementDeserializer
+import no.nav.mulighetsrommet.serialization.json.JsonIgnoreUnknownKeys
+import org.slf4j.LoggerFactory
+import java.util.*
+
+class AmtDeltakerV1TopicConsumer(
+    config: Config,
+    private val deltakere: DeltakerRepository,
+) : KafkaTopicConsumer<UUID, JsonElement>(
+    config,
+    uuidDeserializer(),
+    JsonElementDeserializer()
+) {
+    private val logger = LoggerFactory.getLogger(javaClass)
+
+    override suspend fun consume(key: UUID, message: JsonElement) {
+        val amtDeltaker = JsonIgnoreUnknownKeys.decodeFromJsonElement<AmtDeltakerV1Dto?>(message)
+        when {
+            amtDeltaker == null -> {
+                logger.info("Mottok tombstone for deltaker med id=$key")
```

Comment:
Kunne vi lagt til: `Mottok tombstone for deltaker med id=$key. Sletter deltakeren.`?